product_docs/docs/pgd/5/manual/03-configuring-repositories.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Step 3 - Configuring PGD repositories
+navTitle: Configuring PGD repositories
+deepToC: true
+---
+
+## Configuring PGD repositories
+
+To install and run PGD requires that you configure repositories so that the system can download the appropriate packages.
+
+The following operations should be carried out on each host you wish to be a data node, witness node or subscribe-only node.
```

Comment:
s/`subscribe-only`/`subscriber-only`/

However, this the first time the different node types get mentioned, I'd probably be a bit confused if I were doing this for the first time.

Suggestion:

> The following operations should be carried out on each host. For the purposes of this exercise, each host will be a standard data node, but the procedure would be the same for other node types such as witness or subscriber-only nodes.

Reply:
Text borrowed. thx.